tools/mage/header.yml

```diff
@@ -1,6 +1,7 @@
 rules:
 - include: [.go,.js,.make,Makefile,.styl,.proto]
-  exclude: [.snap,.json,.goreleaser.release.yml,.goreleaser.snapshot.yml]
+  exclude: [.snap,.json,.goreleaser.release.yml,.goreleaser.snapshot.yml,
+    pkg/devicerepository/store/remote/testdata/vendor/foo-vendor]
```

Comment:
Can we glob match `testdata` under `pkg` here?

Reply:
Under the hood this is just a `strings.Contains()` check. Perhaps `testdata` would be enough?

Reply:
I'm fine with that too; the point is to have a pattern here to exclude `testdata`.